micronota/bfillings/tests/test_prodigal.py

```diff
@@ -22,8 +22,10 @@
 class ProdigalTests(TestCase):
     def setUp(self):
         self.temp_dir = tempfile.mkdtemp()
+        self.get_prodigal_path = partial(get_data_path,
+                                         subfolder='data/prodigal')
```

Comment:
I was talking about put 'data' and 'prodigal' into variable instead of just
literals. it doesn't really matter here since they might not be used again.

On Mon, Oct 19, 2015 at 9:14 AM, mortonjt <notifications@github.com> wrote:

> In micronota/bfillings/tests/test_prodigal.py
> <https://github.com/biocore/micronota/pull/22#discussion_r42391548>:
>
> > @@ -22,8 +22,10 @@
> >  class ProdigalTests(TestCase):
> >      def setUp(self):
> >          self.temp_dir = tempfile.mkdtemp()
> > +        self.get_prodigal_path = partial(get_data_path,
> > +                                         subfolder='data/prodigal')
>
> [image: :+1:] for join('data', 'prodigal')
>
> Could you clarify what your mean by "putting dir names into variables"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/22/files#r42391548>.
>